docs/advanced/vm-migration-network.md

```diff
@@ -0,0 +1,204 @@
+---
+sidebar_position: 12
+sidebar_label: VM Migration Network
+title: "VM Migration Network"
+---
+
+A VM migration network is useful for isolating migration traffic from cluster traffic on `mgmt` and other cluster-wide workloads. Using a VM migration network results in better network bandwidth and performance.
+
+:::note
+
+Avoid configuring KubeVirt settings directly, as this can result in unexpected or unwanted system behavior.
+
+:::
+
+## Prerequisites
+
+Before you begin configuring the VM migration network, ensure that the following requirements are met:
+
+- The network switches are correctly configured, and a dedicated VLAN ID is assigned to the VM migration network.
+
+- The [cluster network](../networking/clusternetwork.md) and [VLAN network](../networking/harvester-network.md) are configured correctly. Ensure that both networks cover all nodes and are accessible.
+
+- No virtual machines are being migrated.
+
+- The `ippools.whereabouts.cni.cncf.io` CRD exists. You can check this using the command `kubectl get crd ippools.whereabouts.cni.cncf.io`. In certain [upgrade scenarios](https://github.com/harvester/harvester/issues/3168), the Whereabouts CNI is not installed correctly.
```

Comment:
Not sure if we still need to mention this requirement, since the issue pertains to Harvester 1.x, which is EOL. For now, those CRDs should be installed by default in supported versions. May just remove this check. 

P.S. We may also need to update the storage network documentation. @jillian-maroket